### ISSUE TITLE
clean-css v3

### ISSUE THREAD

**Issue by WADaniel** [carrot/roots-css-pipeline#6]
Any reason for not upgrading `clean-css` to v3?

**Comment by jenius** [comment]
Nope, happy to upgrade to v3. Interested in sending in a pull request for this?

Source maps would need to be an additional options here too. Fairly easy to add, but I can't imagine a use case for this. In development, you shouldn't be minifying, and in production you shouldn't have source maps...

**Comment by WADaniel** [comment]
Yeah, we can give it a try during the week.

Regarding the use case, we want to deploy the same file to development, stage and production. Source maps would help development in this case for us.

**Comment by jenius** [comment]
Sounds good, looking forward to it!

For using the same file for all environments, is there any reason you are trying to do this? With roots, you can simply add a `app.production.coffee` file and change the settings to make sure your builds are slimmed down and minified for production. Shipping source maps to production is not a good idea -- they slow down the page load time, add no benefit, and automatically make your site open source whether that was your intent or not.

**Comment by WADaniel** [comment]
Thank you for your input.

We won't be shipping any source maps to production. We just want to make sure that the pipeline for code is the same in development, stage and production. We've had incidents where code worked in development but not in production and it was due to the pipeline being different and things like "advanced minification" and whatnot was turned on etc.

**Comment by jenius** [comment]
Sure, that makes sense. Source maps however are entirely separate from the code itself so it's pretty close to impossible that not having them would cause any issues with the code, if that makes you feel any better in the meantime!
